### PR TITLE
Add picture of Taylor Otwell and Adam Wathan with Lamborghini

### PR DESCRIPTION
**Pull request by otsch**
Wouldn't that be cool on that page? 😅 Maybe we should ask for permission first 😅

### CODE REVIEW

**Comment by otsch** [comment]
Bildschirmfoto 2022-06-25 um 12 20 03

Here's a screenshot

File: templates/index.php [hunk 1]
```diff
@@ -34,6 +34,12 @@
         is-php-dead.lol
     </h2>
 
+    <div class="w-full md:w-2/3 lg:w-1/2 mx-auto mt-4">
+        <a href="https://twitter.com/taylorotwell/status/788794784932564992" target="_blank" rel="noopener">
+            <img src="/images/php-is-not-dead.jpg" alt="Taylor Otwell and Adam Wathan with Taylor's Lamborghini">
```
Comment (danharrin) [inline templates/index.php:39]: needs more complexity, you should encrypt and decrypt the URL first ;)

**Comment by nurmuhammet-ali** [comment]
I think the car in the picture is McLaren, not the Lamborghini.

File: templates/index.php (hunk 1 above)
Comment (otsch) [reply templates/index.php]: 😜 https://github.com/ryangjchandler/is-php-dead.lol/pull/1/files#diff-5258161aaab90de3916c094462a8b5035d08c02eff481989342aeca0a45c4be5R38

**Comment by otsch** [comment]
> I think the car in the picture is McLaren, not the Lamborghini.

Oh, right. Didn't really pay attention to that 🙈👍🏻

**Comment by nurmuhammet-ali** [comment]
> > I think the car in the picture is McLaren, not the Lamborghini.
> 
> Oh, right. Didn't really pay attention to that 🙈👍🏻

It's okay :)

**Comment by ramsey** [comment]
Your commit messages don't follow the rules for proper commit messages, as defined in the README. 😉

**Pull request closed without merging**